src/app/learn/learn.component.ts

```diff
@@ -6,7 +6,10 @@ import { Component } from '@angular/core';
 })
 export class LearnComponent {
   public sidebarRoutes: any[] = [
-    { name: 'Overview', path: '/learn' },
+    { name: 'Overview', path: '/learn', children: [
+      { name: 'Test1', path: '/learn/test-1' },
+      { name: 'Test2', path: '/learn/test-2' }
+    ]},
```

Comment:
That's the intention, eventually. Even still, that logic wouldn't exist on the component itself, but probably in the layout components. All nav components may now have "children", which will probably be more common than not...